packages/squiggle-lang/__tests__/Distributions/Samples_test.res

```diff
@@ -0,0 +1,46 @@
+open Jest
+open Expect
+
+let makeTest = (~only=false, str, item1, item2) =>
+  only
+    ? Only.test(str, () => expect(item1) -> toEqual(item2))
+    : test(str, () => expect(item1) -> toEqual(item2))
+
+describe("Continuous and discrete splits", () => {
+  makeTest(
+    "check splits one",
+    SampleSet.Internals.T.splitContinuousAndDiscrete([1.432, 1.33455, 2.0]),
+    ([1.432, 1.33455, 2.0], E.FloatFloatMap.empty()),
+  )
+  makeTest(
+    "check splits two",
+    SampleSet.Internals.T.splitContinuousAndDiscrete([
+      1.432,
+      1.33455,
+      2.0,
+      2.0,
+      2.0,
+      2.0,
+    ]) |> (((c, disc)) => (c, disc |> E.FloatFloatMap.toArray)),
```

Comment:
try to use ->, not |>

Reply:
(see my other comment) this is code that was already written and is in my diff because I moved it to a new location.

Reply:
in general I haven't wrapped my head around the `Samples_test`, I think I should work on testing PointSetDist before I do that.